test/karma-conf.js

```diff
@@ -397,7 +397,23 @@ module.exports = function (config) {
             'samples/highcharts/demo/organization-chart/demo.js',
             'samples/highcharts/demo/pareto/demo.js',
             'samples/highcharts/demo/pyramid3d/demo.js',
-            'samples/highcharts/demo/synchronized-charts/demo.js'
+            'samples/highcharts/demo/synchronized-charts/demo.js',
```

Comment:
All these successfully create SVGs on the master using `--reference`. For the renderer samples, the clue is that they don't create a `chart` instance, therefor we look for `document.getElementByTagName('svg')` instead. We used to check for `chart || document.getElementByTagName('svg').length`, but in this PR it has changed to `&&`, which was not the intention.

As for the lates 3 samples, the `/maps/` ones, they load data from Google Spreadsheet. This also works in the master, both for reference and for comparison.

Reply:
This was a messup on my side. This is now fixed.

As as sidenote all the visual tests will be retried until there's a chart or svg found. For most tests this is really not necessary and we should consider making a separate and more explicit test template. However, for some of the maps/ samples that first loads after the (geo)json data has been retrieved it is necessary to retry (or we need to change the demos accordingly so that they more easily can be incorporated in a test).